rust/kernel/iopoll.rs

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Helper routines for polling IO addresses.
+//!
+//! C header: [`include/linux/iopoll.h`](../../../../include/linux/iopoll.h)
+
+use core::mem::MaybeUninit;
+
+use crate::{
+    bindings, c_types,
+    error::{Error, Result},
+    might_sleep_if,
+};
+
+/// read_poll_timeout - Periodically poll an address until a condition is
+///             met or a timeout occurs
```

Comment:
Avoid rewriting the name of the function, also please end sentences with a period:

```suggestion
/// Periodically poll an address until a condition is met or a timeout occurs.
```

Reply:
ack, I will apply it to other cases too